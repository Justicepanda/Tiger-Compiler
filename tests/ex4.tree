$
	<tiger-program>
		let
		<declaration-segment>
			<type-declaration-list>
			<var-declaration-list>
				<var-declaration>
					var
