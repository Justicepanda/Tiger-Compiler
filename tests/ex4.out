LET VAR
Lexical error (line: 2): "_" does not begin a valid token.
ID COLON ID SEMI IN ID ASSIGN INTLIT SEMI ID LPAREN ID RPAREN SEMI END
Tiger-Compiler: Compilation unsuccessful.